acnportal/acnsim/network/charging_network.py

```diff
@@ -1,17 +1,26 @@
-from .constraint_set import ConstraintSet
-
+from .current import Current
+import pandas as pd
+import numpy as np
 
 class ChargingNetwork:
     """
     The ChargingNetwork class describes the infrastructure of the charging network with
     information about the types of the charging station_schedule.
     """
 
-    def __init__(self):
+    def __init__(self, constraints=None, magnitudes=None):
         self._EVSEs = {}
-        self.constraint_set = ConstraintSet()
+        self.constraints = constraints if constraints is not None else pd.DataFrame()
+        self.magnitudes = magnitudes if magnitudes is not None else pd.Series()
+        # Matrix of constraints
+        self.constraint_matrix = None
+        # Vector of limiting magnitudes
+        self.magnitude_vector = None
+        # Dict of constraint_id to row index in constraint matrix
+        self.constraint_index = {}
         self._voltages = {}
```

Comment:
Should voltages also be a Series for consistency?

Reply:
We'll change voltages, phase angles, and magnitudes to numpy arrays, but if they are accessed outside of `charging_network` (for example through `@property` methods), we'll return them as dicts.